python/triqs_dft_tools/converters/wannier90.py

```diff
@@ -315,15 +322,23 @@ def convert_dft_input(self):
                               'symm_op', 'n_shells', 'shells', 'n_corr_shells', 'corr_shells', 'use_rotations', 'rot_mat',
                               'rot_mat_time_inv', 'n_reps', 'dim_reps', 'T', 'n_orbitals', 'proj_mat', 'bz_weights', 'hopping',
                               'n_inequiv_shells', 'corr_to_inequiv', 'inequiv_to_corr', 'kpt_weights', 'kpts', 'dft_code']
+                if wan_centres is not None:
+                    things_to_save.append('wan_centres')
                 if self.bloch_basis:
-                    np.append(things_to_save, 'kpt_basis')
+                    things_to_save.append('kpt_basis')
```

Comment:
Seems like `test/python/w90_convert_SrVO3.py` fails at `h5diff` (line 63) since `kpt_basis` is not present in `_blochbasis.ref.h5`

Reply:
no wonder the old code `np.append(things_to_save, 'kpt_basis')` will not alter `things_to_save` it will only give back a new np.array with `kpt_basis` added. `kpt_basis` was never saved. I updated the reference h5 file. Thanks for finding this bug!